packages/frontend/components/VideoBoard.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState, useRef, useEffect } from 'react'
+import { v4 as uuidv4 } from 'uuid'
+import VideoList from './VideoList'
+import styles from '../styles/videoBoard.module.scss'
+
+export default function VideoBoard() {
+  const [videos, setVideos] = useState([])
+  const videoCreatorRef = React.useRef<HTMLInputElement>(null)
+  const videoTitleRef = React.useRef<HTMLInputElement>(null)
+  const videoLinkRef = React.useRef<HTMLInputElement>(null)
+
+  function uploadVideo(e) {
+    const videoCreator: string = videoCreatorRef.current.value
+    const videoLink: string = videoLinkRef.current.value
+    const videoTitle: string = videoTitleRef.current.value
+
+    if (videoCreator === '' || videoLink === '' || videoTitle === '') return
+    setVideos((prevVideos) => [
+      ...prevVideos,
+      {
+        id: uuidv4(),
+        creator: videoCreator,
+        title: videoTitle,
+        url: videoLink,
+      },
+    ])
+    videoCreatorRef.current.value = null
+    videoLinkRef.current.value = null
+    videoTitleRef.current.value = null
+  }
+
+  return (
+    <>
+      <h2>Welcome to the Video Board</h2>
+      <h3>Share new Video!</h3>
+      <p>Creator Name:</p>
+      <input ref={videoCreatorRef} type="text" />
+      <p>Video Title:</p>
+      <input ref={videoTitleRef} />
+      <p>Video Link:</p>
+      <input ref={videoLinkRef} type="url" />
+      <br />
+      <button onClick={uploadVideo}>Submit!</button>
+      <br />
+      <br />
+      <h3>Videos for Coco</h3>
```

Comment:
see above. If anything this component isn't even needed since the only thing we'll be using from it is the VideoList

Reply:
so basically the entire file can go?

Reply:
removed file